config/publick8s_issue-redirect-jenkins-io.yaml

```
@@ -0,0 +1,58 @@
+ingress:
+  enabled: true
+  className: public-nginx
+
+  image:
+    repository: dockerhubmirror.azurecr.io/jenkinsciinfra/issue-redirect
+    tag: 1 # ?
```

```suggestion
    tag: 0.0.2
```

Freshly released with your changes